package.json

```diff
@@ -22,10 +22,14 @@
   "scripts": {
     "clean": "rm -rf cache out",
     "build": "forge build",
+    "deploy": "forge create Resolver --rpc-url $RPC_URL --private-key $PRIVATE_KEY --constructor-args 0xbD75f629A22Dc1ceD33dDA0b68c546A1c035c458",
```

Comment:
What is this contract? Should be in '.env' instead

Reply:
It's the EAS in the arbitrum network, it's public. Should I still put in the .env?

Reply:
Yes, no problem in case we need to deploy on the mainnet instead of the testnet, the address will be different, if its set in env we can choose where we deploy without the need of a new commit 😄